draft-ietf-lake-authz.md

```diff
@@ -632,6 +641,189 @@ If present, V decrypts and verifies opaque_state as received from W. If that ver
+
+We present this variant as a set of changes to the regular protocol flow, so that U can run as the EDHOC Responder.
+The changes include:
+
+* Voucher_Info and Voucher are transported in EDHOC message_2 and message_3, respectively (instead of message_1 and message_2).
+* The EAD_2 and EAD_3 fields are identified with labels TBD1 and TBD2, respectively.
+* The VREQ / VRES protocol takes place between message_2 and message_3.
+* The Voucher_Request carries G_Y instead of G_X, and the transcript hash TH_2 instead of the hash H_message_1.
+
+~~~~~~~~~~~ aasvg
++-----+-----+                  +-----------+
+|     U     |                  |     V     |
+| Initiator |                  | Responder |
++-----+-----+                  +-----+-----+
+      |                              |
+      |       Trigger Message        |
+      +- - - - - - - - - - - - - - ->|
+      |                              |
+      |       EDHOC message_1        |
+      +<-----------------------------|
+      |                              |
+      |                              |
+      |       EDHOC message_2        |                              W
+      +----------------------------->|                              |
+      | EAD_2 = (TBD1, Voucher_info) |                              |
+      |                              |             VREQ             |
+      |                              +----------------------------->|
+      |                              | (SS, G_X, Voucher_Info,      |
+      |                              |  H_message_1, ?opaque_state) |
+      |                              |                              |
+      |                              |             VRES             |
+      |                              +<---------------------------->|
+      |                              |    (Voucher, ?opaque_state)
+      |     EDHOC message_3          |
+      +<-----------------------------|
+      |   EAD_3 = (TBD2, Voucher)    |
+      |                              |
+      |                              |
+~~~~~~~~~~~
+{: #fig-u-responder title="ELA when U is the EDHOC Responder." artwork-align="center"}
+
+The following subsections detail how the processing changes in each of the three security sessions.
+
+Unless otherwise specified, the processing is the same as described in other subsections of {{protocol}}.
+
+#### Reverse U <-> W {#reverse-u-w}
+
+The protocol between U and W is carried between U and V in message_2 and message_3, and between V and W in the Voucher Request/Response ({{V-W}}).
+
+Voucher Info:
+
+* The EAD_2 item has ead_label = TBD1 and ead_value = Voucher_Info.
+
+Voucher:
+
+* H_handshake is the transcript hash TH_2, sent by V as part of the voucher request, see {{reverse-v-w}}.
+
+#### Reverse U <-> V {#reverse-u-v}
+
+Message 1:
+
+* V composes message_1 and sents it to U.
+* U processes message_1 and extracts SS.
```

Comment:
A more general note here: V has no idea what ciphersuite W and U have pre-agreed to. It may therefore select a cipher suite that is different from the one selected by W and U. In this situation, U needs to reject the proposed cipher suite and start EDHOC session anew, as described in RFC 9528. In this draft, we clearly need to have a consideration somewhere on cipher suite negotiation and the assumption we make in the protocol that whatever cipher suite is used between U and W must also be used by V.

Reply:
Thanks for the comment. Tracking it in #46.